src/zope/app/apidoc/codemodule/directives.rst

```diff
@@ -37,13 +37,13 @@ and the root module is available:
   [(u'zope', 'zope')]
 
 
-The `apidoc:importModule` Directive
------------------------------------
+The ``apidoc:importModule`` Directive
+=====================================
 
-The `importModule` directive allows you to set the
+The ``importModule`` directive allows you to set the
 ``__import_unknown_modules__`` flag of the class registry. When this flag is
-set to false, paths will only be looked up in ``sys.modules``. When set true,
-and the ``sus.modules`` lookup fails, the import function of the class
+set to false, paths will only be looked up in :data:`sys.modules`. When set true,
+and the :data:`sus.modules` lookup fails, the import function of the class
```

Comment:
Hehe. The squiggle underline of a misspelled word made `sys` and `sus` look identical in my font.